layouts/partials/blocks/by-the-numbers.html

```diff
@@ -0,0 +1,494 @@
+<div id="by-the-numbers-wrapper">
+  <div id="numbers-bar">
+    <div class="numbers-bar-item" id="first-div">
+      <div class="bar-content">
+        <div class="circle">
+          <span id="first-circle" class="circle-text">35</span>
+        </div>
+        Policies Changed
+      </div>
+    </div>
+    <div class="divider"></div>
+    <div class="numbers-bar-item" id="second-div">
+      <div class="bar-content">
+        <div class="circle">
+          <span id="second-circle" class="circle-text">14</span>
+        </div>
+        Bills Introduced
+      </div>
+    </div>
+    <div class="divider"></div>
+    <div class="numbers-bar-item" id="third-div">
+      <div class="bar-content">
+        <div class="circle">
+          <span id="third-circle" class="circle-text">8</span>
+        </div>
+        Legal Victories
+      </div>
+    </div>
+  </div>
+
+  <div id="numbers-page" class="pb-16">
+    <div class="grid-container">
+      <div class="impact-numbers-title">
+        <div>Journalism that</div>
+        <div class="spotlight">
+          <span class="spotlight-text">Gets Results</span>
+        </div>
+      </div>
+      <div class="impact-block" id="first-impact-block">
+        <div class="impact-row">
+          <div class="impact-symbol">
+            <svg aria-hidden="true" style="fill: #ffcb03;">
+              {{ partial "helper/svg.html" "envelope" }}
+            </svg>
+          </div>
+        </div>
+        <div class="impact-row symbol-number">3,000+</div>
+        <div class="impact-row impact-text">Reader Tips</div>
+      </div>
+
+      <div class="impact-numbers-graf">
+        Spotlight PA's investigative and public-service journalism changes laws
+        and lives. Our reporting drives government accountability and
+        transparency at all levels, provides an independent check on power and
+        influence, and has saved Pennsylvania taxpayers millions of dollars.
+      </div>
+      <div class="impact-block" id="second-impact-block">
+        <div class="impact-row">
+          <div class="impact-symbol">
+            <svg aria-hidden="true" style="fill: #ffcb03;">
+              {{ partial "helper/svg.html" "gavel" }}
+            </svg>
+          </div>
+        </div>
+        <div class="impact-row symbol-number">1,400+</div>
+        <div class="impact-row impact-text">Right to Know Requests</div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<style>
```

Comment:
I do want to take the time to convert this to Tailwind, just because we're using Tailwind everywhere else.

I found a site https://transform.tools/css-to-tailwind that will give us a first pass at the conversion. For each rule, you'd just find the Tailwind equivalent and then substitute that for the class name. So, `id="by-the-numbers-wrapper` would be `class="font-serif"` `id="numbers-bar"` would be `class="bg-yellow h-[110px] flex w-full items-center` etc. The media query ones will be a little harder because Tailwind classes start as mobile and change as you get larger instead of starting as desktop and changing as you get smaller, but it should be doable.

```css